EmpireLegends/Player.h

```diff
@@ -73,15 +74,22 @@ class Player
 	int MoveOverLand(Territory<Region>*from, Territory<Region>*to, GameMap * map);
 	void PlaceNewArmies(int number, Territory<Region>*destination);
 	void MoveArmies(int number, Territory<Region>*from, Territory<Region>*to, GameMap * map);
-	void DestroyArmy(int number, Territory<Region>*location);
+	void DestroyArmy(int number, Territory<Region>*location, vector<Player*> currentPlayers);
 	void BuildCity(Territory<Region>*territory);
-
+	int getNumofcards();
+	void setNumofcards();
+	vector<Good*> getListGoods();
+	void addGoods(Good* addedGood);
 	Resources* getResources() const;
+	string AndOrAction(Card* cardTwoAction);
 
 private:
 	string playerName;
-	vector<Territory<Region>> playerTerritories;
+	vector<Territory<Region>*> playerTerritories;
 	Hand* playerHand;
+	int NumCards;
+	vector<Good*> playerGoods;
```

Comment:
we don't need this vector, since goods are inside the cards that the player already has